window/src/app/data.service.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { TestBed } from '@angular/core/testing';
+
+import { DataService } from './data.service';
+
+describe('DataService', () => {
+  beforeEach(() => TestBed.configureTestingModule({}));
+
+  it('should be created', () => {
```

Comment:
try and add some real tests

Reply:
Can I merge this then work on tests because this is quite a big blocker for every other ticket in our backlog?